packages/vue-ui/src/Accordion/SubPane.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div class="sub-pane">
+    <slot></slot>
+  </div>
+</template>
+
+<script lang="ts">
+export default {
+  name: "SubPane",
+};
+</script>
+
+<style scoped>
```

Comment:
Where are the SubPane's props?

Reply:
@opichon, props of SubPane component are extracted in Accordion component. And SubPane component just displays the slots passed. It is done similarly in react accordion as well.

Reply:
@uddhab dai, yes. It works that way. You can check out demo for these.